### DISCUSSION TITLE
How effective is SO with only metadata(FortiGate syslog)?

### DISCUSSION THREAD

**Discussion by scolardc** [Security-Onion-Solutions/securityonion#13726] [2.4] [answered]
### Version

2.4.100

### Installation Method

Security Onion ISO image

### Description

other (please provide detail below)

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

8

### RAM

32

### Storage for /

100GB

### Storage for /nsm

200G

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

How effective is SO when only parsing metadata?
Instead of using a network / span port would SO still be effective if I directed the firewall logs (FortiGate syslog) to SO?

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by dougburks** [comment, marked as answer]
From https://docs.securityonion.net/en/2.4/additional-network.html:
> In the [Network Visibility](https://docs.securityonion.net/en/2.4/network.html#network) section, we looked at network visibility provided by Security Onion itself. The ideal situation would be to have Security Onion network sensors covering each and every one of your network segments. If you’re able to achieve that ideal situation, then you may not need any additional network visibility. However, there may be times when you simply can’t cover certain network segments with Security Onion network sensors and that’s when these additional options can be beneficial. Keep in mind, though, that the data that they provide is nowhere near as comprehensive as a full Security Onion network sensor. One option would be [NetFlow](https://docs.securityonion.net/en/2.4/netflow.html#netflow) logs from firewalls, switches, or routers showing what traffic was observed by the network device. Another option would be firewall logs showing what traffic was allowed through the firewall and what traffic was denied. An example of that would be [pfSense](https://docs.securityonion.net/en/2.4/pfsense.html#pfsense) firewall logs. You can find other firewall integrations in the [Third Party Integrations](https://docs.securityonion.net/en/2.4/third-party-integrations.html#third-party-integrations) section.

**Reply by scolardc** [reply]
Thanks for the direction @dougburks.
